dataminer-overview/Legal/DaaS_Planned_Maintenance_Policy.md

```diff
@@ -0,0 +1,98 @@
+---
+uid: DaaS_Planned_Maintenance_Policy
+---
+
+# DaaS (DataMiner as a Service) Planned Maintenance Policy
+
+This policy outlines the procedures and guidelines for planned maintenance on DaaS (DataMiner as a Service). As part of our commitment to providing a secure, reliable, and high-performing service, we will conduct regular maintenance to apply system updates and security patches. This policy ensures that maintenance activities are predictable, transparent, and minimally disruptive to our users.
+
+## Maintenance window
+
+Planned maintenance will be conducted monthly during a predefined maintenance window.
+
+- Scheduled maintenance window: During the second week of each month.
+
+## Scope of maintenance
+
+The monthly maintenance will include the following activities:
+
+- Operating system updates: Applying the latest operating system security patches and updates to ensure a secure environment.
+
+- Security patches: Implementation of critical security patches to mitigate vulnerabilities and potential threats.
+
+- DataMiner software updates:
+
+  - Main Release (current CU): The DataMiner software will be updated to the latest Cumulative Update (CU).
+  - Feature Release (latest version): For users subscribed to the feature release, we will update to the latest available feature release version of DataMiner.
+
+## Communication
+
+We will ensure transparent communication regarding maintenance activities, including any potential system unavailability:
+
+- Notification in advance: Users will receive a maintenance notification at least [X days/weeks] in advance of the scheduled maintenance window. This notification will include details such as:
+
+  - The date and time of the maintenance window.
+  - An outline of the planned updates and changes.
+  - Any anticipated system downtime or unavailability.
+  - Unexpected delays: If there are any unexpected delays or issues that extend the maintenance window, we will notify users immediately and provide revised estimates for service restoration.
```

Comment:
To make this last item stand on its own, so that it also applies for routine maintenance, I'd add a new line above it and remove the indent:
```suggestion

- Unexpected delays: If there are any unexpected delays or issues that extend the maintenance window, we will notify users immediately and provide revised estimates for service restoration.
```